services/movie.py

```diff
@@ -0,0 +1,41 @@
+from typing import List
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: List[int] = None,
+        actors_ids: List[int] = None
+) -> QuerySet:
+    movies = Movie.objects.all()
+
+    if genres_ids is not None:
+        movies = movies.filter(genres__id__in=genres_ids)
+
+    if actors_ids is not None:
+        movies = movies.filter(actors__id__in=actors_ids)
+
+    return movies
+
+
+def get_movie_by_id(movie_id: int) -> QuerySet:
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(
+        movie_title: str,
+        movie_description: str,
+        genres_ids: list[int] = None,

```

Comment:
it accepts Optional[list[int]] because None is default parameter. Fix in all places

Reply:
With all types